middleware/healthcheck/healthcheck.go

```diff
@@ -40,11 +41,19 @@ func New(config ...Config) fiber.Handler {
 			return c.Next()
 		}
 
-		switch c.Path() {
-		case cfg.ReadinessEndpoint:
-			return isReadyHandler(c)
-		case cfg.LivenessEndpoint:
-			return isLiveHandler(c)
+		prefixCount := len(utils.TrimRight(c.Route().Path, '/'))
```

Comment:
Why do we use c.Route().Path here but c.Path() below?

Reply:
because we want to subtract the current route path from the request route